google/cloud/spanner_dbapi/parse_utils.py

```diff
@@ -185,6 +183,8 @@ def classify_stmt(query):
     :rtype: str
     :returns: The query type name.
     """
+    query = sqlparse.format(query, strip_comments=True).strip()
```

Comment:
As this is an external library, it might be good to add a comment here that it has been verified that this library supports exactly the types of comments that Cloud Spanner supports, and no other comment types. That is, the library will strip the following types of comments (from what I can see in the source code of the library this holds true):
- `# single line comment starting with a dash`
- `-- single line comment starting with two hyphens`
- `/* multi line comment */

And it does not support for example the PostgreSQL style dollar quoted string constants (https://www.postgresql.org/docs/current/sql-syntax-lexical.html) (or if it does, document that the parsing would fail if a SQL string contains a comment inside what would be interpreted as a dollar quoted string)